pkgs/top-level/all-packages.nix

```diff
@@ -13835,7 +13835,12 @@ let
 
   jags = callPackage ../applications/science/math/jags { };
 
-  liblapack = callPackage ../development/libraries/science/math/liblapack { };
+  liblapack = callPackage ../development/libraries/science/math/liblapack { 
+    atlas = null;
+  };
```

Comment:
This changes the default version of `liblapack` used in Nixpkgs so far to one that does not have ATLAS. How will that affect other software that uses liblapack, like R and haskell-hmatrix?